Makefile

```diff
@@ -35,22 +35,22 @@ installcheck:
 	@echo ${SOURCES}
 
 clean:
+	@echo "# Not implemented"
 	@echo "erase what has been buit (opposite of make all)"
-	rm -rf droid
 
 dist:
 	@echo "# Not implemented"
 	@echo "create PACKAGE-VERSION.tar.gz"
 
-droid:
-	@echo "# Compacting for Android"
+dist-android:
+	@echo "Compacting for Android"
 	mkdir droid
-	sed 's|HOME/.cache|WD|' main.sh > droid/batti.sh
+	sed 's/.HOME.*\///' main.sh > droid/batti.sh
 	chmod +x droid/batti.sh
-	./main.sh -d > droid/batti.sch
```

Comment:
It seems like you are not *syncing* your *repo*. You got the merge the new changes before doing further work.
If you have trouble handeling multiple `remote` `repos`, I have given you access to the repository. Create `dev-sujit` branch and directly work on it. 

# WHY I'M SAYING THIS?

`master` *should* be stable branch, we test in `dev` and *push* it to `master`.